packages/ndla-ui/src/Programme/Programme.tsx

```diff
@@ -63,9 +64,10 @@ const SubjectsWrapper = styled.div`
 type Props = GradesProps & {
   heading?: string;
   image?: string;
+  messageBox?: boolean;
 };
 
-export const Programme = ({ heading, image, grades }: Props) => {
+export const Programme = ({ heading, image, grades, messageBox }: Props) => {
   const [showGradeIndex, setShowGradeIndex] = useState(0);
```

Comment:
Du finner importen til useTranslation i andre filer.
```suggestion
export const Programme = ({ heading, image, grades, messageBox }: Props) => {
  const { t } = useTranslation();
  const [showGradeIndex, setShowGradeIndex] = useState(0);
```